apps/community/src/app/lab/components/LabCard.tsx

```diff
@@ -0,0 +1,28 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import type { Lab } from '../mocks/labs';
+import * as styles from './LabCard.css';
```

Comment:
```suggestion
import type { Lab } from '~/mocks/labs';
import * as styles from '~/components/LabCard.css';
```

파일명 컨벤션을 정할 필요가 있어보입니다!
LabCard lab-card